app/public/javascripts/uiheader.js

```diff
@@ -27,16 +27,19 @@ $(document).ready(function(){
     $('.modal').modal();
     $('.timepicker').timepicker();
     $('.datepicker').datepicker({minDate: new Date()});
+    $('.notif_button').click(function(){
+        toggleNotification($(this).attr('id')); 
```

Comment:
Right here, make sure to pre-process the attribute id, and grab index 2 of the array. Refer to the later comment for client.ejs of why you want to do this.